packages/good-design/src/apps/goodid/screens/DisputeThanks.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { Center, VStack, Text } from "native-base";
+import { Image } from "../../../core/images";
+import { GoodButton } from "../../../core/buttons";
+import { Title } from "../../../core/layout";
+import { useWizard } from "react-use-wizard";
+
+import BillyDispute from "../../../assets/images/billy-dispute.png";
+
+export const DisputeThanks = () => {
+  const { nextStep } = useWizard();
+
+  return (
+    <VStack space={200} width={343}>
```

Comment:
no fixed widths for screens
screens should be responsive. talk with Sana.

Reply:
right, yes I should update that everywhere

Reply:
fixed here: https://github.com/GoodDollar/GoodWeb3-Mono/pull/154/commits/c02014646b7929a06f22603d8e9860004a4cb56c